### PR TITLE
Fix vulnerabilities in amd64_anax_k8s image

### LINKED ISSUE #4174: Fix vulnerabilities in amd64_anax_k8s:2.31.0-1591 found by Aquasec
Aquasec scan found the following high severity vulnerabilities in amd64_anax_k8s:2.31.0-1591:
- CVE-2024-24786
- CVE-2024-3596

### PR DESCRIPTION
**Pull request by K-Pomian**
## Description

The PR includes fixed for vulnerabilities listed in the below issue.

Fixes #4174 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

Could not test it, because I need an image with the changes to be built by GH Actions pipeline in order to be able to run Aquasec scan on it.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings
- [x] I have tagged the reviewers in a comment below incase my pull request is ready for a review
- [x] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

**Comment by LiilyZhang** [comment]
Changes looks good, please change the commit message to this format:
`Issue open-horizon#4174 - Fix vulnerabilities in amd64_anax_k8s:2.31.0-1591 found by Aquasec`

File: anax-in-k8s/Dockerfile.ubi.amd64 [hunk 1]
```diff
@@ -11,6 +11,7 @@ LABEL description="The agent in a container that is used solely for the purpose
 ARG REQUIRED_RPMS="openssl ca-certificates shadow-utils jq iptables vim-minimal psmisc procps-ng tar"
 RUN  microdnf update  -y --nodocs --setopt=install_weak_deps=0 --disableplugin=subscription-manager \
   && microdnf install -y --nodocs --setopt=install_weak_deps=0 --disableplugin=subscription-manager ${REQUIRED_RPMS} \
+  && microdnf upgrade -y --nodocs --setopt=install_weak_deps=0 --disableplugin=subscription-manager krb5-libs \
```
Comment (omordyk) [inline anax-in-k8s/Dockerfile.ubi.amd64:14]: I guess you should call upgrade before install.This ensures that your system’s dependencies are up-to-date, which can prevent issues with installing new packages.

Comment (K-Pomian) [reply anax-in-k8s/Dockerfile.ubi.amd64:14]: I think that it does not matter, `upgrade` will bump everything it finds that can be upgraded

**Comment by K-Pomian** [comment]
Closing, reason in #4172

**Pull request closed without merging**